src/renderer/components/ft-playlist-add-video-prompt/ft-playlist-add-video-prompt.js

```diff
@@ -44,6 +45,8 @@ export default defineComponent({
       lastShownAt: Date.now(),
       lastActiveElement: null,
       sortBy: SORT_BY_VALUES.LatestUpdatedFirst,
+      keyboardShortcutHandler: (e) => ctrlFHandler(e, this.$refs.searchBar),
```

Comment:
Could you please move all of these into the methods sections, seems more fitting there :)

Reply:
Sorry, they were not working as lambdas in the `methods` section (was causing an issue with `$refs` being `undefined`, fixed now

Reply:
Yep, you can't use lambdas in `methods` because they inherit their parent's `this` as they don't have their own, so Vue can't overwrite it to point to the component.

> Arrow functions differ in their handling of this: they inherit this from the parent scope at the time they are defined. This behavior makes arrow functions particularly useful for callbacks and preserving context. However, arrow functions do not have their own this binding. Therefore, their this value cannot be set by bind(), apply() or call() methods, nor does it point to the current object in object methods.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this